src/phase0/state_transition/epoch_processing.rs

```diff
@@ -179,8 +179,28 @@ pub fn process_justification_and_finalization<
         MAX_VALIDATORS_PER_COMMITTEE,
         PENDING_ATTESTATIONS_BOUND,
     >,
-    _context: &Context,
+    context: &Context,
 ) -> Result<(), Error> {
+    // Initial FFG checkpoint values have a `0x00` stub for `root`.
+    // Skip FFG updates in the first two epochs to avoid corner cases that might result in modifying this stub.
+    if get_current_epoch(state, context) > GENESIS_EPOCH + 1 {
```

Comment:
Noting the spec uses an early return:
```
if get_current_epoch(state) <= GENESIS_EPOCH + 1:
     return
```
...and I implemented the other side of this inequality as I wasn't sure if trying to do the python way is ideal here.

Reply:
i think it is easier to follow if we stick to the spec

i also prefer to keep the bodies of various blocks smaller rather than larger (which the spec does here)